Resources/Textures/Objects/Specific/Hydroponics/cocoa.rsi/meta.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13",
```

Comment:
These sprites don't look at all like the file on /vg/: https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/icons/obj/hydroponics/cocoapod.dmi

Where did you get them from?